specification/analysisservices/resource-manager/readme.md

```diff
@@ -26,10 +26,28 @@ These are the global settings for the AnalysisServices API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2016-05
+tag: package-2017-07

```

Comment:
Could you change package-2017-07 to package-2017-08-beta so it refers to latest and #1542 can be closed.